tools/flang1/flang1exe/dump.c

```diff
@@ -693,7 +693,7 @@ putoptype(char *s, int optype)
 void
 dastli(int astli)
 {
-  int a;
+  unsigned int a;
```

Comment:
would it be easier to do `static_cast` to silence a warning in the `for` loop condition below? Once you change signedness of this variable, this changes should be reflected further, namely, `%d` should be replaced with `%u` in `sprintf` (not sure if it was done wrong for the results of those ASTLI* macors already).

Reply:
Actually this change was wrong. The type of `ASTLI_NEXT` is `int` and isn't the cause of the warning. The warning is caused by the comparison with `astb.astli.stg_avail` which is declared unsigned by the `STG_MEMBERS` macro. As I mentioned previously, solving the `STG_MEMBERS` problem will have to wait for another time. I will undo this hunk.